Src/Common.props

```diff
@@ -25,6 +25,13 @@
     <PackageIconUrl>https://raw.githubusercontent.com/AutoFixture/AutoFixture/79c882c3f4af3cf52ad43e5c95851f25d217ac17/AutoFixtureLogo200x200.png</PackageIconUrl>
     <PackageIcon>icon.png</PackageIcon>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
+
+    <!--Prevents restoring the latest version of FSharp.Core (i.e. FSharp.Core 5.0.0)-->
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+
+    <!--Prevents failing cross target NuGet package restores-->
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```

Comment:
Could you please elaborate a bit what exactly it changes and why we need it? Usually ItJustWorks without this prop 🤔

Reply:
@zvirja as far as I can tell, there's a regression in the NuGet CLI deployed on AppVeyor. It manifests in projects that are cross targeting `netcoreapp1.x`. There's more details in this NuGet [issue](https://github.com/NuGet/Home/issues/7414).